src/nls/it/strings.js

```diff
@@ -85,11 +85,11 @@ define({
     "ERROR_INVALID_SHORTCUTS"           : "Queste scorciatoie non sono valide: {0}",
     "ERROR_NONEXISTENT_COMMANDS"        : "Stai assegnando scorciatoie a comandi inesistenti: {0}",
 
-    
     // Application preferences corrupt error strings
     "ERROR_PREFS_CORRUPT_TITLE"         : "Errore nella lettura delle preferenze",
     "ERROR_PREFS_CORRUPT"               : "Le tue preferenze non sono in un file JSON valido. Il file verrà aperto in modo da poter correggere il formato. Dovrai riavviare {APP_NAME} così le modifiche avranno effetto.",
-
+    "ERROR_PROJ_PREFS_CORRUPT"          : "Il file delle preferenze di progetto non è valido JSON. Il file verrà aperto in modo da poter correggere il formato. Avrete bisogno di ricaricare il progetto affinché le modifiche abbiano effetto.",
```

Comment:
I think it would be better `Le preferenze del progetto non sono in un file JSON valido. Il file verrà aperto ...` (Similar to line 90)